safesql/src/main/java/org/dhatim/safesql/SafeSqlBuilder.java

```diff
@@ -8,31 +8,45 @@
 import java.util.stream.Stream;
 
 public class SafeSqlBuilder implements SafeSqlizable {
-    
+
     static class Position {
-        
+
         private final int sqlPosition;
         private final int paramPosition;
-        
+
         private Position(int sqlPosition, int paramPosition) {
             this.sqlPosition = sqlPosition;
             this.paramPosition = paramPosition;
         }
-        
+
     }
-    
+
     private static final char[] HEX_CODE = "0123456789ABCDEF".toCharArray();
-    
+
     private static final SafeSql DEFAULT_SEPARATOR = SafeSqlUtils.fromConstant(", ");
 
-    private final StringBuilder sqlBuilder = new StringBuilder();
-    private final List<Object> parameters = new ArrayList<>();
-    
+    private final StringBuilder sqlBuilder;
+    private final List<Object> parameters;
+
+    public SafeSqlBuilder() {
+        this.sqlBuilder = new StringBuilder();
+        this.parameters = new ArrayList<>();
+    }
+
+    private SafeSqlBuilder(SafeSqlBuilder other) {
```

Comment:
OK. I was thinking also about a constructor taking a string to initialize the query. Because I often have to do:
```java
SafeSqlBuilder qb = new SafeSqlBuilder();
qb.append("SELECT ...");
```